espr/src/ir/type_ref.rs

```diff
@@ -76,7 +78,7 @@ impl TypeRef {
     pub fn is_simple(&self) -> bool {
         match self {
             TypeRef::SimpleType(..) => true,
-            TypeRef::Named { is_simple, .. } => *is_simple,
+            TypeRef::Named { is_enumerate, .. } => *is_enumerate,
```

Comment:
This is not acceptable without renaming the function `is_simple()` to `is_enumerate`.